97.md

```diff
@@ -0,0 +1,25 @@
+NIP-97
+======
+
+Nostr Login
+-----------
+
+`draft` `optional`
+
+This NIP defines a method for a way for users to prove to apps that they control a certain private key. This can be used to enable login for services like the management panel of image hosts.
```

Comment:
This NIP defines a login flow where apps can request other apps to prove the user controls a private key. The first app displays a link or a QR code with a `nostr+login:` URI. The app with login keys receives that information, signs a NIP-98 payload, and sends it back to the website of the first app, authenticating the user.